fiftyone/server/mutation.py

```diff
@@ -462,10 +462,10 @@ async def search_select_fields(
 
         res = []
         try:
-            schema = view.get_field_schema(flat=True)
+            schema = dataset.get_field_schema(flat=True)
             has_frames = dataset.media_type == "video"
             if has_frames:
-                schema = view.get_frame_field_schema(flat=True)
+                schema = dataset.get_frame_field_schema(flat=True)
```

Comment:
Based on the discussions I've been a part of, the dataset schema should never be used? Is this necesary?